parsl/providers/cobalt/cobalt.py

```diff
@@ -228,3 +228,7 @@ def cancel(self, job_ids):
             rets = [False for i in job_ids]
 
         return rets
+
+    @property
```

Comment:
Could be moved to the parent ClusterProvider

Reply:
In practice, probably. In theory, it seems like not something that should be abstracted away.